mantidproject/mantid
# Indirect Iqt Bug with Monte Carlo Error calculation

**Description of work.**
This PR fixes a bug causing the Monte Carlo errors calculated on Iqt to be false. The problem was caused by identical simulated workspaces!
 Make sure that `Plot error bars on output plots` is turned on in the Indirect Settings GUI (accessible from the bottom left of any indirect interface).

**To test:**
1. `Interfaces`->`Indirect`->`Data Analysis`->`Iqt` tab
2. Load the data below
3. Click `Run`
4. Click `Plot Spectrum` in the output options. The external plot should look like the following graph where you can see the errors are getting bigger towards the end of the spectra:
![image](https://user-images.githubusercontent.com/40830825/59934687-0a1efa00-9444-11e9-8a5c-b3f18b55c96c.png)

**Data Files**
[irs26176_graphite002_red.zip](https://github.com/mantidproject/mantid/files/3315171/irs26176_graphite002_red.zip)
[irs26173_graphite002_res.zip](https://github.com/mantidproject/mantid/files/3315172/irs26173_graphite002_res.zip)


Fixes #25806 and #25819 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
